python/awaazde/base.py

```diff
@@ -46,7 +46,7 @@ def list(self, **kwargs):
         """
         data = {}
         if kwargs:
-            data = {'params': kwargs}
+            data = {'params': kwargs.pop('params')}
```

Comment:
@ashwini-ad Can't we directly pass params in the kwargs, I didn't see any need to pass it as a params key in kwargs argument.

Also can you please test what happens if we pass an empty dictionary in params of her request, If that works then you can also remove this if condition?